### ISSUE TITLE
No `workerHelpers.worker.js` generated & `ReferenceError: Window is not defined`

### ISSUE THREAD

**Issue by laurelch** [RReverser/wasm-bindgen-rayon#16]
Initially, I was receiving the following error 
```
Failed to load module script: The server responded with a non-JavaScript MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.
```

from the `.../pkg/snippets/wasm-bindgen-rayon-[hash]/src/workerHelpers.worker.js?worker_file&type=module` in the browser.

After investigating, I realized there is no `workerHelpers.worker.js` generated in the `/pkg/snippets/wasm-bindgen-rayon-[hash]/src/` of my project. It only contained `workerHelpers.js`.

I attempted to compile both the `demo/` and `test/` directories inside the `wasm-bindgen-rayon` workspace, and in those cases, the `workerHelpers.worker.js` file was generated successfully. However, when I tried to create a standalone project, the file was not generated, even when I just copied the `test/` directory outside of the workspace (and updated its dependencies).

The command I was using is `$ wasm-pack build --target web`, and I did not encounter any errors.

Thank you so much for any guidance!

**Comment by Jonarod** [comment]
+1 Exactly in the same situation...

**Comment by Jonarod** [comment]
Got around the error by copy-pasting this file:

[https://github.com/RReverser/wasm-bindgen-rayon/blob/main/src/workerHelpers.worker.js](https://github.com/RReverser/wasm-bindgen-rayon/blob/main/src/workerHelpers.worker.js)

But I am not able to get passed `await initThreadPool(navigator.hardwareConcurrency);` which never resolves...

**Comment by laurelch** [comment]
Yes, seems that copy-pasting the file helped (partially). After refresh, I'm not getting this error any more
```
Uncaught (in promise) RangeError: Maximum call stack size exceeded
```
but I'm getting this error instead
```
ReferenceError: Window is not defined
```

**Comment by aW4KeNiNG** [comment]
Same problem here

**Comment by TheFrozenFire** [comment]
https://github.com/rustwasm/wasm-bindgen/issues/4233#issuecomment-2463023836

Revert to wasm-bindgen 0.2.93 to fix. The current latest version 0.2.95 has a regression which breaks `link_to!`, as used in https://github.com/RReverser/wasm-bindgen-rayon/blob/09ade9bc064402df85e4b8ac9791ea5866a898ba/src/lib.rs#L59

**Comment by codeart1st** [comment]
https://github.com/rustwasm/wasm-pack/pull/1443

**Comment by officergaff** [comment]
+1 I also got around by copy pasting the https://github.com/RReverser/wasm-bindgen-rayon/blob/main/src/workerHelpers.worker.js
file

**Comment by RReverser** [comment]
I believe this is the same issue as #17, which should hopefully be fixed in 1.3.0 now - please open a new issue if not.
